app/controllers/dashboard_controller.rb

```diff
@@ -2,5 +2,8 @@ class DashboardController < ApplicationController
   before_action :authorize!
 
   def show
+    @user = current_user
+    @query = GithubQuery.new(@user)
```

Comment:
I'm thinking about user vs current_user here. You could pass current_user on line 6 and then also in your view use current_user.method instead of @user.method.